README.md

```diff
@@ -71,6 +71,7 @@ This library currently supports the following cryptocurrencies and address forma
  - RSK (checksummed-hex)
  - RVN (base58check P2PKH and P2SH)
  - SOL (base58, no check)
+ - SRM (base58)
```

Comment:
Please change this to `SRM (base58, no check)` as it is exactly same as SOL

Reply:
Done.